base.r

```diff
@@ -2,6 +2,8 @@ library(rstan)
 library(data.table)
 library(lubridate)
 library(gdata)
+library(dplyr)
+library(tidyr)
```

Comment:
do we need to add these into environment.yml?

Reply:
yes we need to